src/commands/android/index.ts

```diff
@@ -420,6 +420,9 @@ export class AndroidSetup {
 
   async getSetupConfigs(options: Options): Promise<SetupConfigs> {
     const configs = this.getConfigFromOptions(options);
+    if (this.options.standalone) {
+      configs.browsers = 'none';
```

Comment:
We should allow users to pass the `--browser` flag in the standalone mode as well and only set browsers to none if no `--browser` flag is supplied in CLI.